From b1c8a3c70ae37188d0da491678bed081b3037913 Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Tue, 11 Jun 2019 17:48:57 +0000
Subject: [PATCH] MANTA-4039 binder load balancer never met a packet it didn't
 queue Reviewed by: Alex Wilson <alex.wilson@joyent.com> Approved by: Alex
 Wilson <alex.wilson@joyent.com>

---
 deps/mname-balancer | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/mname-balancer b/deps/mname-balancer
index e17d6c4..1f9bbdb 160000
--- a/deps/mname-balancer
+++ b/deps/mname-balancer
@@ -1 +1 @@
-Subproject commit e17d6c493141abbededee49371adddcf65887804
+Subproject commit 1f9bbdbab411aecce21d4ec4f9b8862b199521d1
-- 
2.21.0

